workers/unity/Packages/io.improbable.gdk.tools/LocalLaunch.cs

```diff
@@ -199,13 +209,28 @@ private static string GetClientLogFileFullPath()
             }
         }
 
-        public static void LaunchLocalDeployment()
+        public static void ConfigureLocalDeployment()
         {
             BuildConfig();
 
             var command = Common.SpatialBinary;
             var commandArgs = "local launch --enable_pre_run_check=false";
 
+            LaunchLocalDeployment(command, commandArgs);
+        }
+
+        public static void ConfigureLocalDeploymentWithCustomSnapshot()
+        {
+            BuildConfig();
+
+            var command = Common.SpatialBinary;
+            var commandArgs = "local launch --enable_pre_run_check=false --snapshot " + gdkToolsConfig.CustomSnapshotPath;
+
+            LaunchLocalDeployment(command, commandArgs);
+        }
+
+        private static void LaunchLocalDeployment(string command, string commandArgs)
```

Comment:
The `command` will never change, so why have it as a parameter?
The `commandArgs` will always be snapshot, so maybe just have this pass in an optional path to snapshot?